debian/control

```diff
@@ -25,7 +25,7 @@ Depends: ${gir:Depends},
          gir1.2-eosmetrics-0,
          gir1.2-glib-2.0,
          gir1.2-gtk-3.0,
-         gir1.2-webkit2-3.0,
+         gir1.2-webkit2-4.0,
```

Comment:
FWIW, I believe `${gir:Depends}` should fill all of these in. You can run a local build and then look at debian/eos-social.substvars.